services/order-service/app/main.py

```
@@ -0,0 +1,92 @@
+
+    if settings.debug:
+        await init_db()
+
+    # NATS 커넥션 초기화 — 실패해도 서비스 기동은 계속 (이벤트 발행 best-effort)
+    try:
+        import nats as nats_lib
+
+        nc = await nats_lib.connect(settings.nats_url, connect_timeout=300)
+        set_nats_client(nc)
+        logger.info("NATS 커넥션 연결 완료", nats_url=settings.nats_url)
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)